src/Microsoft.Framework.FileSystemGlobbing/Matcher.cs

```diff
@@ -10,22 +10,22 @@ namespace Microsoft.Framework.FileSystemGlobbing
 {
     public class Matcher
     {
-        private IList<IPattern> _includePatterns = new List<IPattern>();
-        private IList<IPattern> _excludePatterns = new List<IPattern>();
+        protected IList<IPattern> _includePatterns = new List<IPattern>();
+        protected IList<IPattern> _excludePatterns = new List<IPattern>();
```

Comment:
Should we just go ahead and make these properties? Seems strange to have protected fields with _ prefix naming.